src/components/App.js

```diff
@@ -1,15 +1,21 @@
 import React from 'react';
+import '../styles/common.scss';
+import Header from "./Header";
+import Main from './Main';
+import Footer from './Footer';
+import ErrorBoundary from './common/ErrorBoundary';
 
-class App extends React.Component {
-    constructor(props) {
-        super(props);
-        this.greeting = 'Hello World';
-    }
-
+class App extends React.PureComponent {
     render() {
-        return <p>
-            {this.greeting} - from ReactComponent
-        </p>;
+        return (
+            <React.Fragment>
```

Comment:
is there really a need for additional wrapper (`<React.Fragment>`)?

Reply:
Fixed, thank you.